R/calcITransChar.R

```diff
@@ -0,0 +1,102 @@
+#' calcITransChar
+#'
+#' Use IRF data to derive OPENPROM input parameter iTransChar
+#'
+#' @return  OPENPROM input data iTransChar
+#'
+#' @author Anastasis Giannousakis, Fotis Sioutas
+#'
+#' @examples
+#' \dontrun{
+#' a <- calcOutput(type = "ITransChar", aggregate = FALSE)
+#' }
+#'
+#' @importFrom dplyr %>% select
+#' @importFrom tidyr pivot_wider
+#' @importFrom quitte as.quitte
+
+calcITransChar <- function() {
+  
+  a <- readSource("IRF", subtype = "total-van,-pickup,-lorry-and-road-tractor-traffic")
+  #million motor vehicle km/yr
+  KM_VEH_TRUCK <- a*1000
+  #Thousands km/yr
+  a2 <- readSource("IRF", subtype = "passenger-car-traffic")
+  #motor vehicle km/yr
+  KM_VEH <- a2/1000
+  #Thousands km/yr
+  
+  getNames(a) <- "KM_VEH"
+  getSets(a) <- c("region", "period", "KM_VEH")
+  getNames(a2) <- "KM_VEH_TRUCK"
+  getSets(a2) <- c("region", "period", "KM_VEH_TRUCK")
+  q1 <- as.quitte(a)
+  q2 <- as.quitte(a2)
+  mat = matrix(NA, nrow(q1)*3, length(q1))
```

Comment:
why the conversion to matrix here? suggestion: use `rbind` and continue working with `quitte` objects

Reply:
ok I used rbind